### PR TITLE
qtox: 20151221 -> 1.2.4, libtoxcore-dev: 20160105 -> 20160131

### PR DESCRIPTION
**Pull request by tg-x**
update qtox to latest release

### CODE REVIEW

File: pkgs/development/libraries/libtoxcore/new-api/default.nix [hunk 1]
```diff
@@ -52,4 +52,3 @@ stdenv.mkDerivation rec {
     platforms = platforms.all;
   };
 }
-
```
Comment (akaWolf) [inline pkgs/development/libraries/libtoxcore/new-api/default.nix]: No need to delete last line at least.

**Comment by otwieracz** [comment]
If you can, please update utox, too.

**Comment by tg-x** [comment]
Alright, I had a look at utox, and did some cleanups there.

**Comment by tg-x** [comment]
qtox travis build failed due to too much log output, it builds here fine

**Comment by zimbatm** [comment]
Nice

**Pull request merged**